pkg/api/types.go

```diff
@@ -1007,6 +1007,23 @@ type PodSecurityContext struct {
 	// Use the host's ipc namespace.
 	// Optional: Default to false.
 	HostIPC bool `json:"hostIPC,omitempty"`
+
+	// Capabilities are the capabilities to add/drop when running containers
+	Capabilities *Capabilities `json:"capabilities,omitempty"`
```

Comment:
@deads2k I wound up modifying the fuzzer to fuzz randomly to nil OR a random bool.